src/Func.h

```diff
@@ -53,7 +53,7 @@ class Stage {
     Internal::Definition definition;
     std::string stage_name;
     std::vector<Var> dim_vars; // Pure Vars of the Function (from the init definition)
-    std::vector<Internal::StorageDim> storage_dims;
+    Internal::FuncSchedule func_schedule;
```

Comment:
Add a comment that this is just a reference to the func_schedule of the owning Function